samples/templates/default-azuredeploy-docker.json

```diff
@@ -324,7 +324,8 @@
                     "properties": {
                         "linuxFxVersion": "[concat('DOCKER|', variables('registryName'), '/', toLower(parameters('fhirVersion')), '_fhir-server',':', parameters('imageTag'))]",
                         "appCommandLine": "azure-fhir-api",
-                        "alwaysOn": true
+                        "alwaysOn": true,
+                        "healthCheckPath": "/health/check"
```

Comment:
@Ivanidzo4ka I am wondering whether this should be part of properties (line 275) instead of resources -> web -> properties.

Reply:
It doesn't work that way. I've run template with proposed changes.